config/environments/test.rb

```diff
@@ -48,6 +48,7 @@
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
+  config.action_mailer.default_url_options = {host: "example.com"}
```

Comment:
this is okey. let's have this hardcoded here since its just for the text. if we don't need the HOST constant, we can remove it then from the constants file?